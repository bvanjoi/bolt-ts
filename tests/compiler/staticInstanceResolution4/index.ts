// From `github.com/microsoft/TypeScript/blob/v5.8.2/tests/cases/compiler/staticInstanceResolution4.ts`, Apache-2.0 License

class A {
  public foo() {}
}

A.foo();
//~^ ERROR: Property 'foo' does not exist on type 'typeof A'.