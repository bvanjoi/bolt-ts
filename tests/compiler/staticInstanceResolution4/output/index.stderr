  × Property 'foo' does not exist on type 'typeof A'.
   ╭─[../../tests/compiler/staticInstanceResolution4/index.ts:7:3]
 7 │ foo
   · ───
   ╰────
